Application/EdFi.Ods.Standard/Artifacts/MsSql/Structure/Ods/Publishing/0020-EXTENSION-Publishing-Tables.sql

```diff
@@ -0,0 +1,20 @@
+-- Table [publishing].[Snapshot] --
+CREATE TABLE [publishing].[Snapshot] (
+    [SnapshotIdentifier] [NVARCHAR](32) NOT NULL,
+    [SnapshotDateTime] [DATETIME2](7) NOT NULL,
+    [Discriminator] [NVARCHAR](128) NULL,
```

Comment:
Does this need a Discriminator column?

Reply:
I decided to take the generated script as-is, but I'll spare you the internal dialog and just remove it.